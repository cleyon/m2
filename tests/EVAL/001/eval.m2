@newcmd showme
@for i 1 3
And a @i@
@next i
@endcmd
@define myfunc showme
@eval @@{myfunc}
