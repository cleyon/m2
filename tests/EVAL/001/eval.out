And a 1
And a 2
And a 3
